errai-codegen/src/main/java/org/jboss/errai/codegen/meta/impl/apt/APTParameter.java

```diff
@@ -49,7 +49,7 @@ public String getName() {
 
   @Override
   public MetaClass getType() {
-    return new APTClass(actualParameterType);
+    return new APTClass(actualParameterType).getErased();
```

Comment:
Do we know if all the MetaParamter implementations return an erased type?

Reply:
Yeah, it's a bit odd.. but GWTParameter and GWTMethod do return erased types. It may not seem ideal or even correct, but our code base expects that, so I think we should just copy the old behavior for now.